OracleWebLogic/samples/12213-webtier-apache/README.md

```diff
@@ -0,0 +1,66 @@
+Example of Apache Web Server with WebLogic 12.2.1.3.0 Plugin
+=====
+This sample shows how to run a load balancer for a WebLogic cluster, from inside a container in the same network as the WebLogic Server cluster.
+
+# How to build image
+Download file **Oracle WebLogic Server Proxy Plugins 12.2.1.3** [fmw_12.2.1.3.0_wlsplugins_Disk1_1of1.zip](http://www.oracle.com/technetwork/middleware/webtier/downloads/index-jsp-156711.html) (see .download file)
+
+You may build this image with:
+
+        $ docker build -t 12213-apache .
+
+or
+
+        $ sh build.sh
+
+If you need to set up HTTP/HTTPS proxies, build the image using script build_internal.sh.
+
+        $ sh build_internal.sh
```

Comment:
This file MUST be removed. You cannot expose internal Oracle hosts publicly. You must change the `build.sh` to expose any proxies set in the host shell to build.

This is used in all the `buildDockerImage.sh` scripts in this repo:

```bash
# Proxy settings
PROXY_SETTINGS=""
if [ "${http_proxy}" != "" ]; then
  PROXY_SETTINGS="$PROXY_SETTINGS --build-arg http_proxy=${http_proxy}"
fi

if [ "${https_proxy}" != "" ]; then
  PROXY_SETTINGS="$PROXY_SETTINGS --build-arg https_proxy=${https_proxy}"
fi

if [ "${ftp_proxy}" != "" ]; then
  PROXY_SETTINGS="$PROXY_SETTINGS --build-arg ftp_proxy=${ftp_proxy}"
fi

if [ "${no_proxy}" != "" ]; then
  PROXY_SETTINGS="$PROXY_SETTINGS --build-arg no_proxy=${no_proxy}"
fi

if [ "$PROXY_SETTINGS" != "" ]; then
  echo "Proxy settings were found and will be used during the build."
fi
```